resources/service.rb

```diff
@@ -66,27 +66,19 @@
 end
 
 action :load do
-  unless loaded
-    execute "hab sup load #{service_name} #{sup_options.join(' ')}"
-  end
+  execute "hab sup load #{service_name} #{sup_options.join(' ')}" unless loaded
```

Comment:
`loaded` doesn't seem to be defined anywhere. You have a function definition for `service_loaded?` towards the end of the file, but it doesn't appear to do anything just yet.